### PR TITLE
Murali created a new Class

### PR DESCRIPTION
**Pull request by muraliUFC**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,11 @@
   <groupId>SeleniumAutomation</groupId>
   <artifactId>SDET</artifactId>
   <version>0.0.1-SNAPSHOT</version>
- <build>
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
```
Comment (KiransJingaade) [inline pom.xml:7]: looks good to me

File: pom.xml [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+    <build>
```
Comment (KiransJingaade) [inline pom.xml:10]: looks good

File: src/test/java/arrays/MuraliDemo.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package arrays;
+
+public class MuraliDemo {
+    public static void main(String[] args) {
+        System.out.println("Demo to kiran- The Elon Musk");
```
Comment (KiransJingaade) [inline src/test/java/arrays/MuraliDemo.java]: please change the name kiran

File: pom.xml (hunk 1 above)
Comment (muraliUFC) [reply pom.xml:7]: Thanks

File: src/test/java/arrays/MuraliDemo.java (hunk 3 above)
Comment (muraliUFC) [reply src/test/java/arrays/MuraliDemo.java]: Ok i will change

**Review by KiransJingaade — APPROVED** [review]

**Pull request merged**